src/dev/jest/config.js

```diff
@@ -52,7 +52,6 @@ export default {
     '<rootDir>/src/dev/jest/setup/babel_polyfill.js',
     '<rootDir>/src/dev/jest/setup/polyfills.js',
     '<rootDir>/src/dev/jest/setup/enzyme.js',
-    '<rootDir>/src/dev/jest/setup/throw_on_console_error.js',
```

Comment:
Do we know what this was previously used for?

Reply:
It looks like this was used as a way to force tests to fail if console errors were used. But I believe that this is really a question for specific tests not all the tests. It should be done in a particular test if that is a valid assertion in a test.

Reply:
I agree with that reasoning